diff --git a/Sources/ExeDock/Resources/SkinTemplate/skins.js b/Sources/ExeDock/Resources/SkinTemplate/skins.js
index dcd6d73f565f3e065e44e41d37ecb3eebcddfa18..0ea9c06998e74752833954cf97f51e67bd0c8cb8 100644
--- a/Sources/ExeDock/Resources/SkinTemplate/skins.js
+++ b/Sources/ExeDock/Resources/SkinTemplate/skins.js
@@ -83,6 +83,10 @@ const badge = (gm, customLabel) => {
     const isAllCaps = customLabel === customLabel.toUpperCase();
     return `<span class="badge badge-mac">${isAllCaps ? 'MAC' : 'Mac'}</span>`;
   }
+  if (gm.windows) {
+    const isAllCaps = customLabel === customLabel.toUpperCase();
+    return `<span class="badge badge-windows">${isAllCaps ? 'WINDOWS' : 'Windows'}</span>`;
+  }
   return '';
 };
 
